src/domain/ledger/fee-reimbursement.ts

```diff
@@ -1,9 +1,15 @@
 import { toSats } from "@domain/bitcoin"
 
+import { FeeDifferenceError } from "./errors"
+
 export const FeeReimbursement = (prepaidFee: Satoshis): FeeReimbursement => {
-  const getReimbursement = ({ actualFee }: { actualFee: Satoshis }): Satoshis | null => {
+  const getReimbursement = ({
+    actualFee,
+  }: {
+    actualFee: Satoshis
+  }): Satoshis | FeeDifferenceError => {
     const feeDifference = toSats(prepaidFee - actualFee)
-    if (feeDifference < 0 || feeDifference > prepaidFee) return null
+    if (feeDifference < 0 || feeDifference > prepaidFee) return new FeeDifferenceError()
```

Comment:
This is not an error - it is expected behaviour that there may be no fee reimbursement.
As we don't want to use `null` for absence anymore perhaps we should use `undefined` or `0` - but I don't think it should be an error.

Reply:
fee should not be negative or more than the prepaidFee. this is an error; it would mean something had gone wrong somewhere. 

the only non error is if fee = 0, which mean there is no reimbursement.